dspace/config/log4j2-cli.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Logging configuration for DSpace command line tools -->
+<Configuration strict='true'
+               xmlns='http://logging.apache.org/log4j/2.0/config'>
+
+    <Properties>
+        <!-- Default log file directory for DSpace.
+             NOTE: The value of 'dspace.dir' will be replaced by its value in
+             your configuration when DSpace is installed. -->
+        <Property name='log.dir'>${log4j:configParentLocation}/../log</Property>
+
+        <!-- Log level for all DSpace-specific code (org.dspace.*)
+             Possible values (from most to least info):
+	     DEBUG, INFO, WARN, ERROR, FATAL -->
+        <Property name='loglevel.dspace'>INFO</Property>
+
+        <!-- Log level for other third-party tools/APIs used by DSpace
+             Possible values (from most to least info):
+	     DEBUG, INFO, WARN, ERROR, FATAL -->
+        <Property name='loglevel.other'>INFO</Property>
+    </Properties>
+
+    <Appenders>
+        <!-- A1 is for most DSpace activity -->
+        <Appender name='A1'
+                  filePattern="${log.dir}/dspace-cli.log-%d{yyyy-MM-dd}"
+                  type='RollingFile'
+        >
+            <!-- NOTE: The %equals patterns are providing a default value of "unknown" if "correlationID" or
+                 "requestID" are not currently set in the ThreadContext. -->
+            <Layout type='PatternLayout'
+                    pattern='%d %-5p %equals{%X{correlationID}}{}{unknown} %equals{%X{requestID}}{}{unknown} %c @ %m%n'/>
+            <policies>
+                <policy type='TimeBasedTriggeringPolicy'>yyyy-MM-dd</policy>
+            </policies>
+            <!-- Sample deletion policy:  keep last 30 archived files
+            <DefaultRolloverStrategy>
+                <Delete basePath='${log.dir}'>
+                    <IfFileName glob='dspace-cli.log-*'/>
+                    <IfAccumulatedFileCount exceeds='30'/>
+                </Delete>
+            </DefaultRolloverStrategy>
+            -->
+        </Appender>
+
+        <!-- A2 is for the checksum checker -->
+        <Appender name='A2'
+                  filePattern="${log.dir}/checker.log-%d{yyyy-MM-dd}"
+                  type='RollingFile'
+                  fileName='${log.dir}/checker.log'>
```

Comment:
It seems like checksum checker logs (checker.log) will still be written to from *both* Tomcat (in `log4j2.xml`) AND from CLI (in this file).   So, won't we still get corruption in this `checker.log`?

Since the Checksum checker scripts can ONLY currently be run from CLI, maybe we should **remove** the "checker.log" configs from the main `log4j2.xml` (and leave them in this new `log4j2-cli.xml`)?